tools/accuracy_checker/accuracy_checker/adapters/pose_estimation.py

```diff
@@ -337,3 +337,59 @@ def get_poses(subset, candidate):
         scores = np.array(scores)
 
         return persons_keypoints_x, persons_keypoints_y, persons_keypoints_v, scores
+
+
+class SingleHumanPoseAdapter(Adapter):
+    __provider__ = 'single_human_pose_estimation'
+    prediction_types = (PoseEstimationPrediction, )
+
+    def validate_config(self):
+        super().validate_config(on_extra_argument=ConfigValidator.WARN_ON_EXTRA_ARGUMENT)
+
+    def process(self, raw, identifiers=None, frame_meta=None):
+        result = []
+        raw_outputs = self._extract_predictions(raw, frame_meta)
+
+        heatmaps = np.transpose(raw_outputs[self.output_blob][-1], (1, 2, 0))
+        sum_score = 0
+        sum_score_thr = 0
+        scores = []
+        x_values = []
+        y_values = []
+        num_kp_thr = 0
+        vis = [1] * raw_outputs[self.output_blob].shape[1]
+        for kpt_idx in range(raw_outputs[self.output_blob].shape[1]):
+            score, coord = self.extract_keypoints(heatmaps[:, :, kpt_idx])
+            scores.append(score)
+            x, y = self.affine_transform(coord, frame_meta[0]['rev_trans'])
+            x_values.append(x)
+            y_values.append(y)
+            if score > 0.2:
+                sum_score_thr += score
+                num_kp_thr += 1
+            sum_score += score
+        if num_kp_thr != 0:
+            pose_score = sum_score_thr / num_kp_thr
+        else:
+            pose_score = sum_score / raw_outputs[self.output_blob].shape[1]
+        result.append(PoseEstimationPrediction(identifiers[0], np.array([x_values]),
```

Comment:
I understand that this model will be executed with batch > 1 scarcely, but we prefer to define processing in the cycle over list of identifiers and batch, please look on any other adapters.